src/models/asyncapi.ts

```diff
@@ -31,4 +38,15 @@ export interface AsyncAPIDocumentInterface extends BaseModel<v2.AsyncAPIObject |
   allOperations(): OperationsInterface;
   allMessages(): MessagesInterface;
   allSchemas(): SchemasInterface;
+  allSecuritySchemes(): SecuritySchemesInterface;
+  allServerVariables(): ServerVariablesInterface;
+  allParameters(): ChannelParametersInterface;
+  allCorrelationIds(): CorrelationIdsInterface;
+  allTags(): TagsInterface;
```

Comment:
You are missing `allReplies` https://github.com/asyncapi/parser-api/pull/111/files#diff-9eddf4dc51bf6a0125ca7fb094468ad284112270385c41cebce4f8f0a29620abR23

Reply:
@smoya I'm a little confused about where to add the `allReplies` function. If I add it here, then v2 also needs to implement it. But v2 doesn't have replies. I see three possible approaches:

 1) Remove `allReplies` from `parser-api` since it can't be implemented for v2.
 2) Add it here and return an empty array for the v2 implementation.
 3) Add `allReplies` to v3 only.

 I would appreciate some insight on this.